lib/widgets/component/quick_menu.dart

```diff
@@ -1,19 +1,20 @@
 import 'package:flutter/cupertino.dart';
```

Comment:
I think were gonna have to refactor this class when we implement the menu for all buildings. We need a loop or something. Won't have to worry about it for now